### PR TITLE
standardizing setup.py metadata

### PR DESCRIPTION
**Pull request by pdpinch**
@carsongee or @pwilkins can you take a quick look at this? I tried to use the same values from pylti and edx-sga.

### CODE REVIEW

**Comment by carsongee** [comment]
Looks good, while we are at it though, we should add a long_description. For that, we have typically just converted the README.md into README.rst and included it in MANIFEST.in and setup.py. i.e.
https://github.com/mitodl/PyLmod/blob/master/setup.py#L53-L63
and 
https://github.com/mitodl/PyLmod/blob/master/MANIFEST.in

**Comment by carsongee** [comment]
We should also add some troves: https://pypi.python.org/pypi?%3Aaction=list_classifiers

**Comment by pdpinch** [comment]
Added long_description, troves (same as pyLTI) and MANIFEST.

#25 should be merged first, to get README.rst

File: MANIFEST.in [hunk 1]
```diff
@@ -0,0 +1,3 @@
+include README.rst
+include requirements.txt
+include LICENSE.txt
```
Comment (carsongee) [inline MANIFEST.in]: nit: newline needed

File: setup.py [hunk 2]
```diff
@@ -5,9 +5,16 @@
...
     scripts=['bin/xbundle_convert'],
-    author='Isaac Chuang, Shawn Milochik',
-    author_email='ichuang@mit.edu, milochik@mit.edu',
+    author='MIT ODL Engineering',
+    author_email='odl-engineering@mit.edu',
     description='Converts edX courses between OLX and single-XML formats.',
     url='https://github.com/mitodl/xbundle',
     install_requires=['lxml', 'BeautifulSoup'],
+    license='BSD'
+    long_description=README,
```
Comment (carsongee) [inline setup.py]: This is a python syntax error, you need to load the README into a variable ala: `README = open('README.rst').read()` or more properly:
```python
with open('README.rst') as readme_file:
    README = readme_file.read()
```

Comment (pdpinch) [inline setup.py]: Whoops. This is fixed now.

**Comment by carsongee** [comment]
:loud_sound:

**Pull request merged**